tests/mqtt-frames/test.yaml

```diff
@@ -0,0 +1,52 @@
+requires:
+  min-version: 7
+
+args:
+ - -k none
```

Comment:
nit: README mentions the pcap already exists. Can we try to deduplicate it w the `pcap` key here?